homeassistant/components/sensor/prezzibenzina.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for the PrezziBenzina.it service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.prezzibenzina/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, ATTR_TIME)
+from homeassistant.helpers.config_validation import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['prezzibenzina-py==1.1.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_SERVICE = 'service'
+ATTR_FUEL = 'fuel'
+
+ATTRIBUTION = 'Data provided by PrezziBenzina.it'
+
+CONF_STATION = 'station'
+
+SCAN_INTERVAL = timedelta(minutes=120)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATION): cv.string
+})
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Set up the PrezziBenzina platform."""
+    from prezzibenzina import PrezziBenzinaPy
+
+    station = config[CONF_STATION]
+
+    client = PrezziBenzinaPy()
+    dev = []
+    info = client.get_by_id(station)
+    name = client.get_station_name(station)
+
+    for index, info in enumerate(info):
+        dev.append(PrezziBenzinaSensor(index,
+                                       client,
+                                       station,
+                                       name,
+                                       info['fuel']))
+
+    async_add_entities(dev, True)
+
+
+class PrezziBenzinaSensor(Entity):
+    """Implementation of a PrezziBenzina sensor."""
+
+    def __init__(self, index, client, station, name, ft):
+        """Initialize the sensor."""
+        self._client = client
+        self._index = index
+        self._data = None
+        self._station = station
+        self._name = "{} {}".format(name, ft)
```

Comment:
Added this ability :)

Reply:
I'm wondering if the name needs to have the fuel type name, what are your thoughts about this?

Reply:
This is up to you. Personally I would add it because then you know what you have. Also, there are people who will add multiple sensors for multiple car.

Reply:
Should be fine now